include/vsg/maths/mat2.h

```diff
@@ -13,6 +13,7 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 </editor-fold> */
 
 #include <vsg/maths/vec2.h>
+#include <vsg/maths/vec3.h>
```

Comment:
This one hides a bug! Now I've spotted the problem I've fixed it with commit: fc3fd403d93b1adedbac63de60ac95edf8f5c21a which makes this include inappropriate.